var/spack/repos/builtin/packages/vde/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Vde(AutotoolsPackage):
+    """Ethernet compliant virtual network"""
+
+    homepage = "https://github.com/virtualsquare/vde-2"
+    url = "https://github.com/virtualsquare/vde-2/archive/refs/tags/v2.3.3.tar.gz"
+
+    # maintainers("github_user1", "github_user2")
+
```

Comment:
This boilerplate should be removed.

Reply:
Thanks for that, fixed the style as well.